suppkg_rebuild.py

```diff
@@ -36,7 +36,7 @@ def __init__(self, project):
         self.api = StagingAPI(self.apiurl, self.project)
 
     def get_support_package_list(self, project, repository):
-        f = osc.core.get_buildconfig(self.apiurl, project, repository).splitlines()
+        f = [str(x) for x in osc.core.get_buildconfig(self.apiurl, project, repository).splitlines()]
```

Comment:
no need to do this per line. Just split after converting to string. osc code itself wraps into ` from osc.util.helper import decode_it`

Reply:
or just using `decode_list`?

Reply:
As you wish, I personally would go with `decode_it(get_buildconfig()).splitlines()` (pseudo code of course :)